docs/deployment.md

```diff
@@ -7,38 +7,46 @@ This guide outlines how to deploy the Unity Catalog server.
 ### Prerequisites
 
 - To generate the tarball, run the following command in the source code:
-  ```sh
-  build/sbt createTarball
-  ```
+
+```sh
+build/sbt createTarball
+```
 
 ### Unpacking the tarball
 
 - The tarball generated in the `target` directory can be unpacked using the following command:
-  ```sh
-  tar -xvf unitycatalog-<version>.tar.gz
-  ```
+
+```sh
```

Comment:
@HorizonNet I like the code format being indented as it falls under the bullet points.  Is it possible to modify `markdownlint.yaml` to allow for this?

Reply:
@dennyglee Done. It is possible by default. It just looked off to me. Changed it back to be indented. Please take a look again.

Reply:
Sorry, allow me to clarify - the docs.unitycatalog.io deployment page has the following look:

### Option A: Code block aligned with text of bullet point
<img width="993" alt="Screenshot 2024-10-23 at 22 13 48" src="https://github.com/user-attachments/assets/d05e5717-1831-4df1-be5c-c5b3e0d09edc">

> Notice how the code blocks are indented so they are aligned with the text after the bullet point as those code blocks are associated with that bullet point.


### Option B: Code block aligned with the bullet point itself
In the most recent pull, here is what the same page looks like.
 
<img width="1098" alt="Screenshot 2024-10-23 at 22 14 14" src="https://github.com/user-attachments/assets/d9f24910-cb83-4b46-9119-e737b32705e4">

> Notice how the code block is aligned to the bullet point itself, not the text.  It implies that this is a new line vs. belonging to the previous bullet point.


### Question
1. Does it make sense to have code blocks associated/aligned to the text (Option A) vs. the bullet point (Option B)?  
2. If the former makes sense, can we configure markdownlint to accept this type of formatting? 

> Note, I don't mind if we state that we should do Option B vs. Option A.  It's just that we started with Option A and I wanted us to be deliberate with this decision.

Reply:
Ah, got it. Thanks for the example. Option B feels better for the author, but Option A looks better for the reader. I would propose going with Option A. I've updated the PR accordingly. I also updated the README to include this decision for later.